app/mailers/comment_notification_mailer.rb

```diff
@@ -1,22 +1,39 @@
 class CommentNotificationMailer < ApplicationMailer
-  def email_notification(comment)
+
+  #email_notification
+  def reviewer_notification(comment)
     @comment = comment
     @proposal = comment.proposal
 
     # Email all reviewers of this proposal if notifications is true unless they made the comment
-    bcc = @proposal.ratings.map do |rating|
-      user = rating.user
-      if rating.event_teammate.try(:should_be_notified?) && @comment.user_id != user.id
-        user.email
-      end
+    bcc = @proposal.reviewers.with_notifications.map do |reviewer|
+      reviewer.email if @comment.user_id != reviewer.id
     end.compact
 
     if bcc.any?
       mail_markdown(
                     bcc: bcc,
                     from: @proposal.event.contact_email,
-                    subject: "CFP #{@proposal.event.name}: A comment has been posted on '#{@proposal.title}'")
+                    subject: "#{@proposal.event.name} CFP: New comment on '#{@proposal.title}'")
     end
   end
+
+
+  #comment_notification
+  def speaker_notification(proposal, comment)
+    @proposal = proposal
+    @comment = comment
+
+    to = @proposal.speakers.map do |speaker|
```

Comment:
We're already doing this is the notify callback so pass in the speakers list and remove the map call.